software/digiquiz.yml

```
@@ -0,0 +1,10 @@
+name: Digiquiz
+website_url: https://ladigitale.dev/digiquiz/
+description: An online application to publish content created with H5P (documentation in French).
+licenses:
+  - AGPL-3.0
+platforms:
+  - Nodejs
+tags:
+  - Miscellaneous
+source_code_url: https://codeberg.org/ladigitale/digiquiz
```

Is there some sort of rule about having lines at the end of files, like this? Mainly would remove the red minus sign.
```suggestion
source_code_url: https://codeberg.org/ladigitale/digiquiz

```

---

The end of line is not there when the file is merged. This edit isn't needed.